internal/server/statvar/fetcher/svg_fetcher.go

```diff
@@ -165,7 +168,11 @@ func FetchAllSVG(
 			}
 			result[svg].ChildStatVars = append(
 				result[svg].ChildStatVars,
-				&pb.StatVarGroupNode_ChildSV{Id: sv, DisplayName: name},
+				&pb.StatVarGroupNode_ChildSV{
+					Id:          sv,
+					DisplayName: name,
+					SearchNames: []string{name, description},
```

Comment:
Consider checking that `description` has a value before adding it.

Reply:
done